x-pack/tasks/helpers/globs.js

```diff
@@ -26,9 +26,9 @@ function getPluginPaths(plugins, opts = {}) {
     const publicPaths = `plugins/${publicPath}/**/${testPath}/*.js`;
 
     paths = paths.concat([indexPaths, commonPaths, serverPaths]);
-    if(plugin === 'code') {
-      paths.push(`plugins/${serverPath}/**/${testPath}/*.ts`);
-    }
+    // if(plugin === 'code') {
```

Comment:
Won’t this mean you cannot enable code even if you wanted to?

Reply:
you can still enable code in your config, but I don't think user need to care about the test